assets/prototype/domain/eventEditor/context/DatetimeContext/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { createContext } from 'react';
```

Comment:
I am not sure about the filename swap from `DatetimeProvider` to `DatetimeContext`, since it exports both.
Also noticed that path is `eventEditor/context/DatetimeContext/index.tsx`, so maybe just:
`eventEditor/context/Datetime/index.tsx` ?
p.s.: this suggestion is optional, I just wanted to know the reason behind the decision :)

Reply:
Context includes provider, that's why I named it that way 😄